src/test/scala/com/virtuslab/stacktraces/parser/StacktraceParserTest.scala

```diff
@@ -0,0 +1,40 @@
+package com.virtuslab.stacktraces
+
+import com.virtuslab.stacktraces.parser.StacktraceParser
+import com.virtuslab.stacktraces.core.Stacktraces
+import com.virtuslab.stacktraces.printer.PrettyExceptionPrinter
+
+import org.junit.Test
+import org.junit.Assert._
+
+val stacktraceRaw = """Test com.virtuslab.stacktraces.tests.BasicTest.nestedLambdas failed: java.lang.RuntimeException: error, took 0.002 sec
+    at com.virtuslab.stacktraces.tests.BasicTest$package$.$bang(BasicTest.scala:23)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$1$$anonfun$1$$anonfun$1$$anonfun$1(BasicTest.scala:35)
+    at scala.collection.immutable.List.map(List.scala:246)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$2$$anonfun$2$$anonfun$2(BasicTest.scala:35)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$3$$anonfun$3$$anonfun$adapted$1(BasicTest.scala:36)
+    at scala.collection.immutable.List.flatMap(List.scala:293)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$4$$anonfun$4(BasicTest.scala:36)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$5$$anonfun$adapted$1(BasicTest.scala:37)
+    at scala.collection.immutable.List.map(List.scala:246)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$6(BasicTest.scala:37)
+    at com.virtuslab.stacktraces.tests.BasicTest.$anonfun$adapted$1(BasicTest.scala:38)
+    at scala.collection.StrictOptimizedIterableOps.flatMap(StrictOptimizedIterableOps.scala:117)
+    at scala.collection.StrictOptimizedIterableOps.flatMap$(StrictOptimizedIterableOps.scala:104)
+    at scala.collection.immutable.Range.flatMap(Range.scala:59)
+    at com.virtuslab.stacktraces.tests.BasicTest.nestedLambdas$$anonfun$1(BasicTest.scala:38)
+    at com.virtuslab.stacktraces.util.TestExecutor$.executeTest(TestExecutor.scala:13)
+    at com.virtuslab.stacktraces.tests.BasicTest.nestedLambdas(BasicTest.scala:40)
+    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
+    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
+    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
+    at java.lang.reflect.Method.invoke(Method.java:566)
+"""
+
+class StacktraceParserTest:
+
+  @Test 
+  def parserTest =
+    val stacktrace = StacktraceParser.parse(stacktraceRaw)
+    val prettyStackTrace = Stacktraces.convertToPrettyStackTrace(stacktrace)
+    PrettyExceptionPrinter.printStacktrace(prettyStackTrace)
```

Comment:
For now these tests doesn't assert anything particular, though they can fail, because `convertToPrettyStackTrace` can throw an exception if something goes wrong, which will cause JUnit interpret as a failure.